pkgs/servers/home-assistant/custom-lovelace-modules/weather-chart-card/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "weather-chart-card";
+  version = "2.4.9";
+
+  src = fetchFromGitHub {
+    owner = "mlamberts78";
+    repo = "weather-chart-card";
+    rev = "V${version}";
+    hash = "sha256-1mM5zxBwYUqdoXQyyCJgaih+RlPL+VBWNNbFnErGSco=";
+  };
+
+  npmDepsHash = lib.fakeHash;
```

Comment:
^

Reply:
Yep, that's what I mean with comment above.